pkg/collector/corechecks/checkbase.go

```diff
@@ -44,14 +45,15 @@ func NewCheckBase(name string) CheckBase {
 	return CheckBase{
 		checkName:     name,
 		checkID:       check.ID(name),
+		checkExtraID:  "",
 		checkInterval: check.DefaultCheckInterval,
 	}
 }
 
 // BuildID is to be called by the check's Config() method to generate
 // the unique check ID.
 func (c *CheckBase) BuildID(instance, initConfig integration.Data) {
-	c.checkID = check.BuildID(c.checkName, instance, initConfig)
+	c.checkID = check.BuildID(c.checkName, c.checkExtraID, instance, initConfig)
```

Comment:
is c.checkExtraID already set at this point?

Reply:
BuildID is called after CommonConfigure is called, in core checks like ntp and snmp.
But I get your point on this one. Not sure how to fix it though.